website/templates/project/view_file.mako

```diff
@@ -2,6 +2,28 @@
 
 <div id="alertBar"></div>
 
+<%def name="resource()"><%
```

Comment:
Is it possible to DRY this def up?   I see that the difference here is that you are adding the 'file for a ' to what you send to GA, unlike `project_base.mako` where you just send 'project' or something.  Not sure how hard it is to consolidate these defs - not a mako expert!